### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by johanneslamers** [iainurquhart/FacetedNav_CraftPlugin#1]
Hi @iainurquhart can I use this plugin on a project? It's not listed on Straightupcraft.com. Are you planning to add new features?

**Comment by iainurquhart** [comment]
Yes, by all means use it. I'll probably be extending it further in the next few weeks as I need to use it further than just categories in this next project.

**Comment by johanneslamers** [comment]
Thanks @iainurquhart. If you need same ideas about this plugin for the next release you can give me a call. Have you ever used Reefine for EE? https://devot-ee.com/add-ons/reefine

I think if you add some custom field support, and numeric range, and checkboxes you can sell this plugin from the new craft store. It's a real life saver for most websites.

**Comment by johanneslamers** [comment]
@iainurquhart how can I add custom markup and fields per category? The markup is now all the same for all categories specified in craft.facetedNav.getNav

**Comment by iainurquhart** [comment]
I don't understand your question sorry, if you dump() the return value for getNav you'll see it's just an object containing categories and active filters. There's no restrictions on what you can do there when iterating through the results.
